src/Controller/Component/PaginatorComponent.php

```diff
@@ -319,7 +319,7 @@ public function validateSort(Table $object, array $options)
         }
 
         $inWhitelist = false;
-        if (!empty($options['sortWhitelist'])) {
+        if (isset($options['sortWhitelist']) && $options['sortWhitelist'] !== false) {
```

Comment:
Wouldn't just isset check be enough?  Given that the default behavior is no whitelist I don't see why anyone would bother setting it to `false` to achieve the same.